spec/routing/gpo_verification_routing_spec.rb

```diff
@@ -59,15 +48,11 @@
 
     it 'routes to endpoints controlled by feature flag' do
       get_routes.each do |route|
-        expect(get: route).to be_routable
-      end
-
-      create_routes.each do |route|
-        expect(post: route).to be_routable
```

Comment:
`be_routable` is true even if the route directs to a 404, so this wasn't really testing anything before. We don't have `POST` routes for these paths.